src/org/usfirst/frc4904/robot/commands/PistonIntakeRelease.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsytems.PistonIntake;
+import org.usfirst.frc4904.standard.commands.SingleOp;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class PistonIntakeRelease extends CommandGroup{
+	public PistonIntakeRelease(){
+		requires(RobotMap.Component.pistonIntake);
+		addParallel(new MotorConstant(RobotMap.Component.pistonIntake.roller, 0));
```

Comment:
idk much about this but wouldn't u want to push out the cube with something like an outtake speed?

Reply:
That's what PistonIntakeRollout does. We have two different outtake commands.